src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+    let totalCost = days * dailyRate;
+
+    if (days >= 7) {
+        totalCost -= 50;
```

Comment:
Avoid using literals in expressions. Instead, define constants that explain what these values represent. For example, instead of '7' and '50', use constants like 'LONG_TERM_RENTAL_DAYS' and 'LONG_TERM_DISCOUNT'. This makes the code more readable and maintainable.